src/org/robockets/buttonmanger/buttons/ToggleButton.java

```diff
@@ -0,0 +1,40 @@
+package org.robockets.buttonmanger.buttons;
+
+import org.robockets.buttonmanger.Button;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+public class ToggleButton extends Button {
+	private boolean toggle = false;
+
+	/**
+	 * The command should have some code to run in the run method and other code to run in the interrupted method
```

Comment:
Assuming the run() and interrupted() methods are left to the specific instance of the button?